machine/v1/types_nutanixprovider.go

```diff
@@ -78,6 +79,17 @@ type NutanixMachineProviderConfig struct {
 	// +optional
 	Categories []NutanixCategory `json:"categories"`
 
+	// gpus is a list of GPU devices to add to the machine's VM.
+	// +listType=map
+	// +listMapKey=type
+	// +optional
+	GPUs []NutanixGPU `json:"gpus,omitempty"`
+
+	// dataDisks holds information of the data disks attached to the Machine's VM
+	// +listType=set
```

Comment:
This will need to be validated by webhooks since Machine API specs are not actual APIs

Reply:
Okay, will add the Machine webhooks for the newly added Nutanix provider spec.